src/xo-app/host/tab-network.js

```diff
@@ -147,7 +147,7 @@ class PifItem extends Component {
       <td>
         <Select
           onChange={this._configIp}
-          options={configModes}
+          options={configModes || []}
```

Comment:
Correctly initialize the state or use `utils/EMPTY_ARRAY` to avoid creating a new array at each render.